deployments/multiple-ragapps/docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
       - ragapp-network
   
   manager:
-    image: ragapp/manager
+    image: ragapp/manager:latest
```

Comment:
build update for ragapp missing?

Reply:
hmm, they all have the build config

Reply:
the build path for ragapp container is not changed